base_layer/wallet/src/transaction_service/protocols/transaction_receive_protocol.rs

```diff
@@ -333,12 +333,14 @@ where TBackend: TransactionBackend + 'static
                             break;
                         }
                     },
-                     _ = cancellation_receiver => {
-                        info!(target: LOG_TARGET, "Cancelling Transaction Receive Protocol for TxId: {}", self.id);
-                        return Err(TransactionServiceProtocolError::new(
-                            self.id,
-                            TransactionServiceError::TransactionCancelled,
-                        ));
+                    result = cancellation_receiver => {
```

Comment:
We should probably log the result via `warn!` if it's not `ok()`.

Reply:
This can only happen if the other end of the channel is shutdown in which case all the channels in this select! will do the same and didn't feel like it needs to be logged as it will happen only in shutdown or else all the channels will show the same error (which are logged).